spec/features/user_answers_question.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+describe "answer a question", type: :feature do
+
+  before(:each) do
+    @user_asks_question = user
+    @question = FactoryGirl.create(:question)
+  end
+
+  
+
+
+
+  def user
+    @user ||= FactoryGirl.create(:user).tap do |user|
```

Comment:
It might make sense to create a factory that confirms a user.So you can simply write sometihng like 

`FactoryGirl.create(:user, :confirmed)`